### ISSUE TITLE
input() causes IPython console to encounter an error

### ISSUE THREAD

**Issue by TheRandomFox** [spyder-ide/spyder#17393]
### What steps will reproduce the problem?

IPython ver 7.13.0  
Any instance of the input() function will cause the console to freeze and return with the attached error.

### Traceback
```python-traceback
  File "C:\Users\Rando\Anaconda3\lib\site-packages\qtconsole\base_frontend_mixin.py", line 138, in _dispatch
    handler(msg)
  File "C:\Users\Rando\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole\widgets\debugging.py", line 325, in _handle_input_request
    if self._hidden:
AttributeError: 'ShellWidget' object has no attribute '_hidden'
```

## Versions

* Spyder version: 4.1.3 
* Python version: 3.7.4
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  4.0.0 (OK)
cloudpickle >=0.5.0            :  2.0.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  6.1.0 (OK)
numpydoc >=0.6.0               :  1.2 (OK)
paramiko >=2.4.0               :  2.8.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.8.0 (OK)
pygments >=2.0                 :  2.11.2 (OK)
pylint >=0.25                  :  2.3.1 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  1.0.3 (OK)
qtconsole >=4.6.0              :  5.2.2 (OK)
qtpy >=1.5.0                   :  1.11.2 (OK)
rtree >=0.8.3                  :  0.9.7 (OK)
sphinx >=0.6.6                 :  3.0.4 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  22.3.0 (OK)

# Optional:
cython >=0.21                  :  0.29.25 (OK)
matplotlib >=2.0.0             :  3.5.1 (OK)
numpy >=1.7                    :  1.21.5 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.7.3 (OK)
sympy >=0.7.3                  :  1.9 (OK)
```

**Comment by rhkarls** [comment]
This should be fixed in the latest version (5.2.2) . See https://github.com/spyder-ide/spyder/issues/17373

**Comment by ccordoba12** [comment]
Yep, that's correct. This error is already fixed in our latest version (5.2.2).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

After that, you can connect Spyder to your base environment by following these instructions:

http://docs.spyder-ide.org/current/faq.html#using-existing-environment
